include/crm/common/resources.h

```diff
@@ -291,10 +291,6 @@ typedef struct resource_object_functions_s {
     char *(*parameter)(pcmk_resource_t *rsc, pcmk_node_t *node, gboolean create,
                        const char *name, pcmk_scheduler_t *scheduler);
 
-    // \deprecated Do not use
-    void (*print)(pcmk_resource_t *rsc, const char *pre_text, long options,
-                  void *print_data);
-
```

Comment:
Don't bother with this commit -- I have a PR coming up that takes this entire struct out of the public API at once, and I can drop this when I do that

Reply:
Okay, I'll wait for that PR and then deal with updating this one.